src/components/docs-version-switcher/index.tsx

```diff
@@ -17,6 +17,25 @@ import { DocsVersionSwitcherOption, DocsVersionSwitcherProps } from './types'
 
 const IS_DEV = process.env.NODE_ENV !== 'production'
 
+/**
+ * Construct a project name to be used in ariaLabels for each option.
+ */
+export function setProjectForAriaLabel(
+	projectName,
+	currentRootDocsPath,
+	currentProduct
+) {
+	let projectNameForLabel: string
+	if (projectName) {
+		projectNameForLabel = projectName
+	} else {
+		const docsName = currentRootDocsPath.shortName ?? currentRootDocsPath.name
+		projectNameForLabel = docsName.includes(currentProduct.name)
```

Comment:
this is merely a style thing, but if we also add the variable:
 
 ```js
const prodName = currentProduct.name;
```

i think we could get this ternary to one line.